controllers/tools/views/applications.njk

```diff
@@ -28,7 +51,17 @@
             </p>
 
             <form action="" method="get" class="tools-filter-form">
+                <div class="tools-filter-form__item">
+                    <label for="country" class="ff-label">Country</label>
 
+                    <select name="country" class="ff-text">
```

Comment:
Should this be `ff-select` rather than `ff-text`?

Reply:
Probably, but that class doesn't exist (though it's used in the templates a bit)!

Looks like this without `.ff-text`
<img width="729" alt="image" src="https://user-images.githubusercontent.com/394376/68943723-e61dd580-07a3-11ea-995a-ad18b636fc0e.png">

Reply:
Hah, I guess it's just there by convention then. Go without the class, preferable to using `ff-text` on something unexpected.